src/RepoAuditor/ExecuteModules.py

```diff
@@ -63,7 +111,7 @@ def Execute(
     ignore_warnings_module_names = ignore_warnings_module_names or set()
     max_num_threads = 1 if single_threaded else None
 
-    with dm.Nested("Processing {}...".format(inflect.no("module", len(module_infos)))) as modules_dm:
+    with dm.Nested(f"Processing {inflect.no('module', len(module_infos))}...") as modules_dm:

```

Comment:
nit: From a style perspective, I'm not a fan of embedded anything other than very simple code in f-strings. In this case, I intentionally used format because `inflect.no("module", len(module_infos))` exceeded that threshold. In this particular case, I decided to use format over an f-string because using an f-string makes one alternate the standard quote style to get it to work.